src/main/scala/viper/server/core/DefaultVerificationExecutionContext.scala

```diff
@@ -0,0 +1,41 @@
+package viper.server.core
+
+import java.util.concurrent.{ExecutorService, Executors, Future, ThreadFactory}
+
+import akka.actor.{ActorSystem, BootstrapSetup}
+import akka.actor.setup.ActorSystemSetup
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutorService}
+
+object DefaultVerificationExecutionContext {
+  // at least 2 threads are needed if the actor system is also using the same thread pool (as opposed to creating a
+  // actor system with the default executor like `ActorSystem(actorSystemName)`)
+  val minNumberOfThreads: Int = 2
+}
+
+class DefaultVerificationExecutionContext(actorSystemName: String = "Actor_System", threadNamePrefix: String = "thread") extends VerificationExecutionContext {
```

Comment:
Could you please add a Scaladoc to this file to explain, in 1-2 paragraphs, what is the motivation for this class, and what are the intended use cases?